### PR TITLE
Get rid of user_ldap LogWrapper and use LoggerInterface

### PR DESCRIPTION
**Pull request by come-nc**
user_ldap still used a `LogWrapper` class which was a wrapper around `\OCP\Util::writeLog`.

This PR deletes this class and makes sure `\Psr\Log\LoggerInterface` is used instead as it was already the case in the rest of the app classes.

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]
Looks good :+1:

File: apps/user_ldap/lib/User/User.php [hunk 1]
```diff
@@ -112,24 +112,16 @@ class User {
 	 * @brief constructor, make sure the subclasses call this one!
 	 * @param string $username the internal username
 	 * @param string $dn the LDAP DN
```
Comment (blizzz) [inline apps/user_ldap/lib/User/User.php:114]: then remove the other @param's as well. Might as well $username to $userId to make it more clear, too.

**Review by blizzz — COMMENTED** [review]
:+1:

File: apps/user_ldap/lib/User/User.php (hunk 1 above)
Comment (come-nc) [reply apps/user_ldap/lib/User/User.php:114]: The username and dn params are needed for psalm (and others tools) to know the type. I was not sure if adding string typing to the parameters would have side effects so I left it as docblock. The other parameters are already typed in the prototype so the docblock for them was redundant.

**Review by ChristophWurst — APPROVED** [review]
:broom: :shipit:

**Pull request merged**